cli/multisig.go

```diff
@@ -298,27 +309,29 @@ var msigInspectCmd = &cli.Command{
 					target += " (self)"
 				}
 				targAct, err := api.StateGetActor(ctx, tx.To, types.EmptyTSK)
-				if err != nil {
-					return xerrors.Errorf("failed to resolve 'To' address of multisig transaction %d: %w", txid, err)
-				}
-				method := stmgr.MethodsMap[targAct.Code][tx.Method]
-
 				paramStr := fmt.Sprintf("%x", tx.Params)
-				if decParams && tx.Method != 0 {
-					ptyp := reflect.New(method.Params.Elem()).Interface().(cbg.CBORUnmarshaler)
-					if err := ptyp.UnmarshalCBOR(bytes.NewReader(tx.Params)); err != nil {
-						return xerrors.Errorf("failed to decode parameters of transaction %d: %w", txid, err)
-					}
 
-					b, err := json.Marshal(ptyp)
-					if err != nil {
-						return xerrors.Errorf("could not json marshal parameter type: %w", err)
+				if err != nil {
+					fmt.Fprintf(w, "%d\t%s\t%d\t%s\t%s\t%s(%d)\t%s\n", txid, "pending", len(tx.Approved), target, types.FIL(tx.Value), "new account, unknown method", tx.Method, paramStr)
```

Comment:
instead of saying unkown method in all cases, we should say "Send" if the method is 0